docs/getting-started-guides/mesos.md

```diff
@@ -5,20 +5,23 @@
 Mesos allows dynamic sharing of cluster resources between Kubernetes and other first-class Mesos frameworks such as [Hadoop][1], [Spark][2], and [Chronos][3].
 Mesos also ensures applications from different frameworks running on your cluster are isolated and that resources are allocated fairly.
 
-Running Kubernetes on Mesos allows you to easily move Kubernetes workloads from one cloud provider to another to your own physical datacenter.
+Running Kubernetes on Mesos allows you to easily move Kubernetes workloads from one cloud provider to another or to your own physical datacenter.
```

Comment:
I think this sentence in isolation is not useful unless there are instructions somewhere on how to do it (and it's actually easier than doing the same thing on native Kubernetes).